src/triage/tracking.py

```diff
@@ -0,0 +1,282 @@
+import datetime
+import platform
+import getpass
+import os
+import requests
+import subprocess
+import logging
+from functools import wraps
+from triage.util.db import scoped_session
+from contextlib import contextmanager
+from triage import __version__
+
+
+from triage.component.results_schema import ExperimentRun, ExperimentRunStatus
+
+
+def infer_git_hash():
```

Comment:
Is this something we expect to be able to reliably get? I know some folks have processes for which this would work, but I wonder how common....

It also seems like the kind of thing that folks may want to configure/specify, especially when they're not in such environments -- _e.g._ in AWS Batch this could be read from the process environment.

Reply:
I would expect for many runs this would not be populated. It's just a 'nice to have' column, for when people have checked out a particular branch.

Reply:
Yep. Like I said, I don't know if there would be something sensible that people would otherwise like to use to tag runs – perhaps a (Git) tag specified from the process environment, or a more generic tag specified the same way.

Reply:
We can talk about this at the meeting. I'm trying not to come up with reasons to make the input interface bigger just to support this, but if people really want to be able to inject a tag for runs we can do that. Even if we did, I probably wouldn't want a generic tag to share a column with the maybe-available git hash.